src/content/1.7/release/release-a-new-version-of-prestashop.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Release a new version of PrestaShop
+weight: 4
+---
+
+# Release a new version of PrestaShop
+
+## Overview
+
+What does it mean, "to release a new version of "PrestaShop" ? Actually many different answers are eligible.
+
+You could say that "a new version of PrestaShop is released when there is a new release on the GitHub repository". Or you could say "a new version of PrestaShop is released when I can download a new version on prestashop.com". You could also say "a new version of PrestaShop is released when the auto-upgrade module allows to upgrade to this new version".
+
+So we should first put some words on all of these statements, to avoid any confusion.
+
+When we release a new version on PrestaShop:
+
+1. we create a new Build of PrestaShop, which is a ZIP archive which contains the software
+2. we make it available on GitHub as a GitHub release, and tag the branch used for the build to mark the exact code commit corresponding to this build
+3. we make it available on prestashop.com as a downloadable ZIP archive
+4. we activate multiple settings in PrestaShop Addons system so the PrestaShop API allows the new version of PrestaShop to download modules adapted to it
+5. we activate multiple settings in PrestaShop Internationalization API so it allows the new version of PrestaShop to download localization packs
+6. we update the XML stream being used by auto-upgrade module to monitor what versions of PrestaShop are available
+7. we create new Docker images for the new version
+
+## 1. Make a build
```

Comment:
```suggestion
## 1. (re)Make a build
```
The purpose is also to remake a build of a version from the same tag and check if there are differences, in order to people to be sure that we don't add anything, or that this is repeatable. (trust and confidence)